src/utils.js

```diff
@@ -24,6 +24,8 @@ export const getWikipediaAttrFromUrl = url => {
 
 	addMiniGalleryRow = ( lang, title ) => {
 		requestPageMedia( lang, title, mediaData => {
-			renderPreviewMedia( document, mediaData )
+			if ( mediaData ) {
```

Comment:
the response mediaData can be `[]` empty array
```suggestion
			if ( mediaData && mediaData.length > 1 ) {
```

Reply:
You are correct sir, just tested with an article with no images